api/src/pcapi/core/fraud/api.py

```diff
@@ -300,6 +303,36 @@ def _check_user_eligibility(
     )
 
 
+def is_subscription_name_valid(name: typing.Optional[str]) -> bool:
+    if not name:
+        return False
+    stripped_name = name.strip()
+    return bool(is_latin(name) and stripped_name)
```

Comment:
`is_latin(name) and stripped_name` c'est déjà un bool

Reply:
je suis pas sûr de comprendre l'utilité de `stripped`, c'est pour gérer quel cas ?

Reply:
C'est pour gérer les cas d'espaces seuls du genre ` `

Reply:
du coup on peut écrire 
`return is_latin(name) and name != ""`

Reply:
C'est vraiment quelle que soit la quantité d'espaces utilisée.

Reply:
<img width="425" alt="Capture d’écran 2022-02-24 à 15 28 22" src="https://user-images.githubusercontent.com/44037911/155545629-9cd6dac7-0780-4d44-a814-c53c01cf34d0.png">

Reply:
C'est pour ça que je mets bool